src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+@import 'var';
```

Comment:
The file 'var' is imported here. Ensure that this file exists and contains the necessary variable definitions (like `$mainAccent`, `$secondary`, `$blueAccent`) used in your SCSS files. If it doesn't exist or lacks these definitions, it will cause compilation errors.